src/admin/js/ets-admin.js

```diff
@@ -0,0 +1,37 @@
+jQuery(document).ready(function ($) {
+    //On click on refresh button then ajax call
+    jQuery('#wl-refresh-summary').on('click', function(e) {
+        e.preventDefault();
+        let postId = jQuery(this).data('post-id');
+
+        jQuery.ajax({
```

Comment:
@ravisoni6262 wp.ajax.post calls WP ajax actions for you, no need for many lines of code.